src/components/Auth/Login/Login.react.js

```diff
@@ -128,6 +128,23 @@ class Login extends Component {
             loading: false,
           });
         });
+      if (cookies.get('loggedIn')) {
```

Comment:
Instead of relying on cookie value, you should use the state value. Cookie value are only supposed to be used to hydrate the store. Also, you should shift this code block inside the 
```
if (payload.accepted) {

}
```
block, then you don't even need to check whether he is logged in or not coz if that statement enters the payload. accepted block then technically he is logged in.

Reply:
Good catch, will make the changes